packages/core/src/shared/utilities/textDocumentUtilities.ts

```diff
@@ -165,3 +165,22 @@ export async function createTempFileForDiff(
     await applyChanges(doc, range, code)
     return tempFileUri
 }
+
+/**
+ * Indents the given code based on the current document's indentation at the selection start.
+ *
+ * @param {any} message - The message object containing the code.
+ * @param {vscode.TextDocument} doc - The VSCode document where the code is applied.
```

Comment:
non-blocker: This will make the value actually clickable in vscode when you hover the docstring and also if TextDocument were to be renamed this would automatically update. Note you need to add a string in between "param" and the {} or it doesnt render properly
```suggestion
 * @param - {@link vscode.TextDocument} doc - The VSCode document where the code is applied.
```